packages/components/src/components/domainproperties/dataclasses/DataClassDesigner.tsx

```diff
@@ -157,7 +173,8 @@ export class DataClassDesigner extends React.PureComponent<Props, State> {
                     key={model.domain.domainId || 0}
                     domainIndex={0}
                     domain={model.domain}
-                    headerPrefix={model.name}
+                    headerTitle={'Fields'}
+                    helpTopic={null} // null so that we don't show the "learn more about this tool" link for this domains
```

Comment:
should this value be a prop?

Reply:
For the designers that wrap the DomainForm, we are putting the help topic link in the upper properties panel instead so this null is the way to "turn if off" for the DomainForm component (that should l likely be refactored since this seems to be the common case/usage).